src/support_sphere/lib/presentation/components/auth/login_form.dart

```diff
@@ -142,7 +171,7 @@ class _LoginButton extends StatelessWidget {
         return state.status.isInProgress
             ? const CircularProgressIndicator()
             : ElevatedButton(
-                onPressed: state.isValid
+                onPressed: context.read<LoginCubit>().isLoginButtonEnabled()
```

Comment:
```suggestion
                onPressed: state.isLoginButtonEnabled
```

Once you've added the computed property, you should be able to do the following

Reply:
updated!